concrete/src/Block/Command/AddAliasDefaultsBlockCommandHandler.php

```diff
@@ -18,11 +18,15 @@ public function __invoke(AddAliasDefaultsBlockCommand $command)
                 $b = Block::getByID($command->getOriginalBlockID(), $mc, $command->getOriginalAreaHandle());
                 if ($b) {
                     $bt = $b->getBlockTypeObject();
+                    $displayOrder = $b->getBlockDisplayOrder();
                     if ($bt->isCopiedWhenPropagated()) {
-                        $b->duplicate($page, true);
+                        $b = $b->duplicate($page, true);
+                        $b->setAbsoluteBlockDisplayOrder($displayOrder);
                     } else {
-                        $b->alias($page);
+                        $b->alias($page, $displayOrder);
```

Comment:
This deserves a separate pull request with the associated discussion around it.  Can you revert this and open a separate pull request after I have a chance to merge this?